### PR TITLE
[cmd/opampsupervisor] Use nop components during bootstrapping

### PR DESCRIPTION
**Pull request by dmolenda-sumo**
**Description:** <Describe what has changed.>
Switch the Supervisor's bootstrap config to use nopreceiver and nopexporter

**Link to tracking Issue:** #32455

### CODE REVIEW

**Review by kasia-kujawa — APPROVED** [review]

File: cmd/opampsupervisor/supervisor/templates/bootstrap.yaml [hunk 1]
```diff
@@ -1,11 +1,7 @@
 receivers:
-  otlp:
-    protocols:
-      http:
-        endpoint: "localhost:{{.EndpointPort}}"
```
Comment (evan-bradley) [inline cmd/opampsupervisor/supervisor/templates/bootstrap.yaml:5]: Could you remove `.EndpointPort` and any logic to find a free port from `getBootstrapInfo` in `supervisor.go`?

**Review by evan-bradley — COMMENTED** [review]
Thanks for taking this @dmolenda-sumo. Could you create a changelog entry? You can generate the file with `make chlog-new`.

File: .chloggen/nop-coponents-during-bootstrapping.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: opamp/supervisor
```
Comment (evan-bradley) [inline .chloggen/nop-coponents-during-bootstrapping.yaml:7]:
```suggestion
component: cmd/opampsupervisor
```

File: .chloggen/nop-coponents-during-bootstrapping.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Switch the OpAMP Supervisor's bootstrap config to use nopreceiver and nopexporter
```
Comment (evan-bradley) [inline .chloggen/nop-coponents-during-bootstrapping.yaml:10]:
```suggestion
note: Switch the OpAMP Supervisor's bootstrap config to use the nopreceiver and nopexporter
```

**Review by evan-bradley — APPROVED** [review]

**Pull request merged**